saleor/graphql/account/types.py

```diff
@@ -643,6 +680,18 @@ class Group(ModelObjectType[models.Group]):
             "True, if the currently authenticated user has rights to manage a group."
         ),
     )
+    accessible_channels = NonNullList(
+        Channel,
+        description="List of channels the group has access to."
+        + ADDED_IN_313
+        + PREVIEW_FEATURE,
+    )
+    restricted_access_to_channels = graphene.Boolean(
+        required=True,
+        description="Determine if group have restricted access to channels."
```

Comment:
If this is false, then we have an access to all channels, right? If yes, wouldn't be good to include this in the field description?

Reply:
No, this field will be defined directly in the `PermissionGroup` mutations